src/ralph/settings/base.py

```diff
@@ -431,6 +431,45 @@ def str_to_bool(s: str) -> bool:
     'TO': 'Technical Owner',
 }
 
+# Change management settings
+
+CHANGE_MGMT_OPERATION_STATUSES = {
+    'OPENED': os.getenv(
+        'CHANGE_MGMT_OPERATION_STATUS_OPENED', 'Open'
+    ),
+    'IN_PROGRESS': os.getenv(
+        'CHANGE_MGMT_OPERATION_STATUS_IN_PROGRESS', 'In Progress'
+    ),
+    'RESOLVED': os.getenv(
+        'CHANGE_MGMT_OPERATION_STATUS_RESOLVED', 'Resolved'
+    ),
+    'CLOSED': os.getenv(
+        'CHANGE_MGMT_OPERATION_STATUS_CLOSED', 'Closed'
+    ),
+    'REOPENED': os.getenv(
+        'CHANGE_MGMT_OPERATION_STATUS_REOPENED', 'Reopened'
+    ),
+    'TODO': os.getenv(
+        'CHANGE_MGMT_OPERATION_STATUS_TODO', 'Todo'
+    ),
+    'BLOCKED': os.getenv(
+        'CHANGE_MGMT_OPERATION_STATUS_BLOCKED', 'Blocked'
+    )
+}
+
+CHANGE_MGMT_BO_LOADER = os.getenv('CHANGE_MGMT_BO_LOADER', None)
```

Comment:
i'd change `BO` to `BASE_OBJECT` - `BO` could be misleading here (someone could evaluate it to `BackOffice` for example)